lib/route.js

```diff
@@ -955,6 +955,23 @@ function route(harvester, name, model, schema, routeOptions) {
     }
 
     this.appendLinks = appendLinks;
+
+    function disallow(methods) {
+        _.forEach(methods, function (methodName) {
+            var method = _this[methodName]();
+            method.options.notAllowed = true;
+            method.register();
```

Comment:
Have a look at 

https://github.com/agco/harvesterjs/blob/feature/119-Provision-Permissions-Export-from-Harvester/lib/route.js#L56

`this.get` is defined above and is registered below:

https://github.com/agco/harvesterjs/blob/feature/119-Provision-Permissions-Export-from-Harvester/lib/route.js#L516

Otherwise I would have to add some additional property to options that would not influence the interceptor until `register` method is called.
Also `register` is meant for single method, while `readOnly` and `immutable` operates on multiple methods (within the same route/resource).
btw. this is why i think `register` is a bad idea. Simply changing the properties should be sufficient. (The `disableAuthorization` right now also does not require `register` to be called).